Azure/sonic-buildimage
# [sonic-cfggen]: Updata config db to generate lanes

Signed-off-by: Gang Lv ganglv@microsoft.com



#### Why I did it
Config db schema generated by minigraph can’t pass yang validation, PORT table does not have 'lanes' field.

#### How I did it
Update src/sonic-config-engine/tests/mocktable/config_db.json to add port lanes.

#### How to verify it
Build sonic-config-engine.
Run command 'sonic-cfggen -m xxx.xml --print-data', and check lanes field.

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog

Fix #9553 

#### A picture of a cute animal (not mandatory but encouraged)
